app/components/tabs_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class TabsComponent < ViewComponent::Base
```

Comment:
Hopefully what's happening with this component makes sense. Glad to explain if not though.

This is a pattern I think would be cool to use more as the project grows and we accumulate these repeated layouts and visual blocks. This is a completely "dumb" component in the sense that it knows nothing about the domain. It just easily recreates the tab layout we're using across the app.